src/styles/main.scss

```
@@ -1,3 +1,200 @@
-body {
+body,
+p,
+a,
+h2 {
+  padding: 0;
   margin: 0;
 }
+
+html {
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  font-weight: 400;
+  color: #060b35;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  top: 0;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  padding: 0 50px;
+
+  &__logo {
+    height: 40px;
+    width: 40px;
+    padding-top: 10px;
+  }
+}
+
+.nav {
+  display: flex;
+  justify-content: flex-end;
+  margin: 0;
+  padding: 0;
+  font-weight: 500;
+
+  &__list {
+    display: flex;
+    justify-content: flex-end;
+    margin: 0;
+    padding: 0;
+  }
+
+  &__element {
+    position: relative;
+    list-style-type: none;
+    margin-left: 20px;
+    cursor: pointer;
+  }
+
+  &__link {
+    line-height: 60px;
+    text-transform: uppercase;
+    display: inline-block;
+  
+    &:hover {
+      color: #00acdc;
+    }
+  }
+}
+
+.active {
+  margin-left: 0;
+}
+
+.is-active {
+  color: #00acdc;
+  line-height: 60px;
+  text-transform: uppercase;
+  display: inline-block;
+  
+  &:hover {
+    color: #00acdc;
+  }
+
+  &::after {
+    background-color: #00acdc;
+    position: absolute;
+    bottom: 0;
+    left: 0;
+    content: "";
+    height: 4px;
+    width: 100%;
+    border-radius: 8px;
+  }
+}
+
+.main {
+  display: flex;
+  flex-wrap: wrap;
+  padding: 50px 40px;
+  justify-content: center;
+  box-sizing: border-box;
+}
+
+.card {
+  padding: 16px;
+  width: 200px;
+  box-sizing: border-box;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  align-items: center;
+
+  &__photo {
+    display: flex;
+    height: 134px;
+    width: 160px;
+    padding: 16px 3px 40px;
+  }
+
+  &__description {
+    font-size: 12px;
+    font-weight: 500;
+    margin-bottom: 4px;
+    line-height: 18px;
+  }
+
+  &__code {
+    font-size: 10px;
+    color: #616070;
+    margin-bottom: 16px;
+  }
+
+  &__rating {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 24px;
+  }
+}
+
+.main__card {
+  margin: 0 48px 46px 0;
+}
+
+.main__card:nth-child(4n) {
+  margin-right: 0;
+}
+
+.stars {
+  display: flex;
+
+  &__star {
+    height: 16px;
+    width: 16px;
+    margin-right: 4px;
+    background-repeat: no-repeat;
+    background-position: center;
+    background-image: url(../images/star.svg);
+  }
+}
+
+@for $starCount from 1 through 5 {
+  &--#{$starCount} .stars__star:nth-child(-n + #{$starCount}) {
+    background-image: url(../images/star-active.svg);
+  }
+}
+
+.rating__reviews {
+  font-size: 10px;
+}
+
+.price {
+  display: flex;
+  justify-content: space-between;
+  align-content: center;
+  margin-bottom: 16px;
+
+  &__text {
+    display: flex;
+    align-items: center;
+    font-size: 12px;
+    color: #616070;
```

don't forget to create variables for all colors

```suggestion
    color: $c-color;
```